File: level1/sumOfTwoNum.js
function solution(a, b) {
    let answer = 0;
    let startNum = Math.min(a, b);
    let endNum = startNum === a ? b : a;
    for(let i = startNum; i <= endNum; i++) {
        answer += i;
    }
    return answer;
}

console.log(solution(3, 5))

File: level1JAVA/src/SumOfTwoNum.java
class SumOfTwoNum {
    public long solution(int a, int b) {
        long answer = 0;
        for(int i = Math.min(a, b); i <= Math.max(a, b); i++) {
            answer += i;
        }
        return answer;
    }

    public static void main(String[] args) {
        SumOfTwoNum sumOfTwoNum = new SumOfTwoNum();
        long result = sumOfTwoNum.solution(3, 5);
        System.out.println("result = " + result);
    }
}
